packages/ui/src/components/va-stepper/VaStepper.vue

```diff
@@ -2,10 +2,21 @@
   <div
     class="va-stepper"
     :class="{ 'va-stepper--vertical': $props.vertical }"
+    v-bind="ariaAttributesComputed"
   >
     <ol
       class="va-stepper__navigation"
+      ref="stepperNavigation"
       :class="{ 'va-stepper__navigation--vertical': $props.vertical }"
+
+      @click="onNavigationValueChange()"
+      @keyup.enter="onNavigationValueChange()"
+      @keyup.space="onNavigationValueChange()"
+      @keyup.left="onArrowKeyPress('prev')"
+      @keyup.up="onArrowKeyPress('prev')"
+      @keyup.right="onArrowKeyPress('next')"
+      @keyup.down="onArrowKeyPress('next')"
```

Comment:
Prevent page scrolling.

```suggestion
      @keyup.up.prevent="onArrowKeyPress('prev')"
      @keyup.right="onArrowKeyPress('next')"
      @keyup.down.prevent="onArrowKeyPress('next')"
```

(Accept this suggestion if we need to prevent scrolling with arrow keys)

Reply:
And why keyup, not keydown?